server/src/game/role/psychic.rs

```diff
@@ -78,42 +78,42 @@ impl Psychic {
             .filter(|player_ref|!player_ref.has_innocent_aura(game))
             .collect();
 
-        let Some(selected_evil) = evil_players.choose(&mut rng) else {return ChatMessage::PsychicFailed};
+        let Some(selected_evil) = evil_players.choose(&mut rng) else {return ChatMessageVariant::PsychicFailed};
 
         let random_players: Vec<_> = Psychic::get_valid_players(game, actor_ref).into_iter()
             .filter(|p|p!=selected_evil)
             .filter(|player_ref|!player_ref.has_innocent_aura(game))
             .collect::<Vec<_>>()
             .choose_multiple(&mut rng, 2).copied().collect();
         
-        let Some(random_player0) = random_players.get(0) else {return ChatMessage::PsychicFailed};
-        let Some(random_player1) = random_players.get(1) else {return ChatMessage::PsychicFailed};
+        let Some(random_player0) = random_players.get(0) else {return ChatMessageVariant::PsychicFailed};
+        let Some(random_player1) = random_players.get(1) else {return ChatMessageVariant::PsychicFailed};
 
         let mut out = [selected_evil, random_player0, random_player1];
         out.shuffle(&mut rng);
-        ChatMessage::PsychicEvil { players: [out[0].index(), out[1].index(), out[2].index()] }
+        ChatMessageVariant::PsychicEvil { players: [out[0].index(), out[1].index(), out[2].index()] }
 
     }
-    fn get_psychic_result_good(game: &Game, actor_ref: PlayerReference)->ChatMessage{
+    fn get_psychic_result_good(game: &Game, actor_ref: PlayerReference)->ChatMessageVariant{
```

Comment:
Ideally this would return a better type (as ChatMessageVariant can be literally any chat message). But also this function is and will forever be used only in this one file, so feel free to mark this comment as resolved if you don't care